install-dev/upgrade/sql/1.7.7.1.sql

```diff
@@ -1 +1,9 @@
 /* PHP:ps_1771_update_customer_note(); */;
+
+SET SESSION sql_mode='';
+SET NAMES 'utf8';
+
+/* add id_order in cart */
+ALTER TABLE `PREFIX_cart`
+    ADD `id_order` INT(10) UNSIGNED NULL DEFAULT NULL,
+    ADD INDEX (id_order);
```

Comment:
No need to update the existing cart to add the id_order?

Reply:
Indeed we need a script to update this, or a super doped SQL query if someone knows how to ^^